pom.xml

```diff
@@ -208,16 +215,34 @@
                 <artifactId>maven-jar-plugin</artifactId>
                 <version>2.4</version>
                 <configuration>
-                	<archive>
-                		<manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
-                	</archive>
+                    <archive>
+                        <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+                    </archive>
                     <excludes>
                         <exclude>**/*.java</exclude>
                         <exclude>**/*.html</exclude>
                     </excludes>
                 </configuration>
             </plugin>
          
+            <!-- Creates the source jar -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+                <version>2.1</version>
+                <configuration>
+                    <includePom>true</includePom>
+                </configuration>
+                <executions>
+                    <execution>
+                       <id>attach-sources</id>
+                       <goals>
+                           <goal>jar-no-fork</goal> 
+                       </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            
```

Comment:
Is the configuration in the parent pom not sufficient?
Maybe it just needs a <pluginManagement> here to specify the version?

Reply:
Possibly, though my thinking here was that it was approved in the EE4J_8 branch, so I'll merge it here to have a point to start.